Logger/Logger.cpp

```diff
@@ -0,0 +1,50 @@
+#include <iostream>
+#include <fstream>
+#include "Logger.hpp"
+
+enum exceptionLevel
```

Comment:
ten enum mógłby sie znalezc w osobnym pliku, było by to bardziej eleganckie rozwiazanie